tests/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2016-2017 Bitergia
+# GPLv3 License
+
+FROM debian:stretch-slim
```

Comment:
OK. Changed to:

```
FROM debian:9-slim
```

Reply:
Oooops. When I went to look for this, there is no debian:9-slim, only debian:strecht-slim. So I leave it that way, since this one is much smaller than the other...